### PR TITLE
Completed user can view list of lobbies.

### PR DESCRIPTION
**Pull request by ryancurrie**
image

### CODE REVIEW

File: client/scenes/lobbyList/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+      this.setState({
+        lobbies: lob,
+        lobbiesLoaded: true
+      })
+    })
+  }
+
+  render() {
+    if (this.state.lobbiesLoaded) {
+      console.log(this.state.lobbies)
```
Comment (thebearingedge) [inline client/scenes/lobbyList/index.js]: You can omit this console.log

File: client/scenes/lobbyList/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          <div className="row" style={styles}>
+            <span>Choose a lobby</span>
+          </div>
+          <div className="row">
+            <LobbyList lobbies={this.state.lobbies} />
+          </div>
+        </div>
+      )
+    }
+    else {
```
Comment (thebearingedge) [inline client/scenes/lobbyList/index.js]: It's better to return early in this situation.

```js
if (!this.state.lobbiesLoaded) return null
return (
  // etc.
)

File: server/index.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+const app = express()
+
+app.use(express.static(path.join(__dirname, 'public')))
+
+const server = http.createServer(app)
+const io = IO(server)
+
+const lobbies = []
+
+if (lobbies.length === 0) {
```
Comment (thebearingedge) [inline server/index.js]: `lobbies.length` will always be `0` when the app starts.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by ryancurrie** [comment]
Completed requested changes.

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**